repos/spack_repo/builtin/packages/sedacs/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+from spack_repo.builtin.build_systems.python import PythonPackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
+
+
+class Sedacs(PythonPackage, CudaPackage):
```

Comment:
Have you reviewed the naming scheme (https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#choosing-a-package-name) for python packages before choosing the file name?